hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -668,7 +672,9 @@ private CertificateServer initializeCertificateServer(String clusterID,
             .setRatisServer(scmHAManager.getRatisServer())
             .setCRLSequenceId(getLastSequenceIdForCRL()).build();
 
-    return new DefaultCAServer(subject, clusterID, scmID, certStore);
+    return new DefaultCAServer(subject, clusterID, scmID, certStore,
+        new DefaultProfile(),
```

Comment:
should we start with the DefaultCAProfile?

Reply:
When this PR code is integrated, when 2 CA Servers are started will change this. It will come in next in PR.